package SET_MONSTER;

message CONTENTS
{
	message CHAR_ID_X_Y
	{
		optional int32 id = 1;
		optional bytes name = 2;
		optional int32 x = 3;
		optional int32 y = 4;
		optional int32 lv = 5;
		optional int32 prthp = 6;
		optional int32 maxHp = 7;
		optional int32 power = 8;
	}
	repeated CHAR_ID_X_Y data = 1;
}